<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana_principal</class>
 <widget class="QMainWindow" name="Ventana_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLineEdit" name="line_ingreso_nombre">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Buscar por nombre</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="boton_buscar_pelicula">
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Busqueda de pel√≠culas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="0">
     <widget class="QLineEdit" name="line_ingreso_actor_2">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Buscar actor 2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="line_ingreso_actor_1">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Buscar actor 1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Busqueda por actor/es&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="boton_buscar_actor">
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>263</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_buscar_actor</sender>
   <signal>clicked()</signal>
   <receiver>Ventana_principal</receiver>
   <slot>_VentanaPrincipal__buscar_peliculas_por_actores()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_buscar_pelicula</sender>
   <signal>clicked()</signal>
   <receiver>Ventana_principal</receiver>
   <slot>_VentanaPrincipal__buscar_peliculas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_VentanaPrincipal__buscar_peliculas_por_actores()</slot>
  <slot>_VentanaPrincipal__buscar_peliculas()</slot>
 </slots>
</ui>
